### PR TITLE
Fix scope flags do not take effect for walrus apply/delete

### PR DESCRIPTION
**Pull request by aiwantaozi**
**Problem:**
1. scope flags do not take effect for walrus apply/delete
2. interpolation failed while file path included a dot

**Solution:**
1. use scope from flags while existed
2. update regex for file path

**Related Issue:**
#1764 
#1763

### CODE REVIEW

File: pkg/cli/cmd/apply.go [hunk 1]
```diff
@@ -23,6 +23,11 @@ func Apply(sc *config.Config) (*cobra.Command, error) {
 		Example: manifestExample("apply"),
 		Short:   "Apply a configuration to a resource using the provided file path or folder.",
 		Run: func(cmd *cobra.Command, args []string) {
+			// Set server context.
+			sc.ServerContext = sc.ServerContext.Merge(config.ServerContext{
```
Comment (gitlawr) [inline pkg/cli/cmd/apply.go:27]: Should we move this to PreRun?

File: pkg/cli/cmd/prerun.go [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/seal-io/walrus/pkg/cli/config"
+	"github.com/seal-io/walrus/pkg/cli/manifest"
+)
+
+func preRun(sc *config.Config, opts *manifest.Option) func(*cobra.Command, []string) {
```
Comment (gitlawr) [inline pkg/cli/cmd/prerun.go:10]: rename the funciton to make the intent clear

**Review by gitlawr — APPROVED** [review]
LGTM

**Review by alexcodelf — APPROVED** [review]
LGTM

**Pull request merged**